com.unity.render-pipelines.high-definition/Runtime/Sky/HDRISky/HDRISky.cs

```diff
@@ -169,6 +169,24 @@ public override int GetHashCode()
             return hash;
         }
 
+        /// <summary>
+        /// Determines if the SkySettings is very different from another. This is going to be used to determine whether
+        /// to reset completely the ambient probe instead of using previous one when waiting for current data upon changes.
+        /// In addition to the checks done with the base function, this HDRISky override checks whether the cubemap parameter
+        /// has changed if both settings are HDRISky.
+        /// </summary>
+        /// <param name="otherSettings">The settings to compare with.</param>
+        /// <returns>Whether the settings are deemed very different.</returns>
+        public virtual bool IsVeryDifferentFrom(SkySettings otherSettings)
+        {
+            if (otherSettings.GetSkyRendererType() != GetSkyRendererType())
```

Comment:
Can't we put this part in the base SkySetting implementation?

Reply:
And we should! I added it last minute and I thought I did put it there; moving now.

Reply:
Also made a mistake there, I missed the override keyword.